app/src/main/cpp/skyline/services/pctl/IParentalControlService.cpp

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "IParentalControlService.h"
+
+namespace skyline::service::pctl {
+    IParentalControlService::IParentalControlService(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager, Service::pctl_IParentalControlService, "pctl:IParentalControlService", {
+    }) {}
```

Comment:
Sorry, but I just noticed that this wasn't due to line wrapping but was on the next line. We don't tend to put the `}) {}` on the next line if the list is empty, only if it's being terminated while having 1 or more item.

Reply:
Meeeehhhh, it's easier to have it two lines for the future :(

Reply:
Fine then, I don't mind it all that much :)